src/lib/testing/equivalent.ts

```diff
@@ -226,10 +237,18 @@ function equivalentProvable<
         handleErrors(
           () => f1(...inputs),
           () => f2(...inputWitnesses),
-          (x, y) => Provable.asProver(() => assertEqual(x, to.back(y), label))
+          (x, y) => Provable.asProver(() => assertEqual(x, to.back(y), label)),
+          label
         );
       });
     });
+    if (verbose) {
+      let ms = (performance.now() - start).toFixed(1);
+      let runs = nRuns.toString().padStart(2, ' ');
+      console.log(
+        `${label.padEnd(20, ' ')}    success on ${runs} runs in ${ms}ms.`
+      );
+    }
```

Comment:
new argument `verbose` prints a nice summary of the test run. I used this to debug why the new EC unit tests were taking so long (turns out that `scale()` just takes a pretty long time, like a few seconds, I'm planning to look into how it can be made faster, as a follow-up)